src/Networking/v2/Extensions/Layer3/Models/FloatingIp.php

```diff
@@ -70,6 +70,8 @@ public function delete()
 
     public function retrieve()
     {
-        $this->executeWithState($this->api->getFloatingIp());
+        $response = $this->execute($this->api->getFloatingIp(),
```

Comment:
Shouldn't this equivalent to the following?

```php
$response =  $this->executeWithState($this->api->getFloatingIp());
```

Reply:
Yes, looks like it is same. I guided myself more by other models in this project, for example:

https://github.com/php-opencloud/openstack/blob/master/src/Networking/v2/Models/Network.php#L53
https://github.com/php-opencloud/openstack/blob/master/src/Networking/v2/Models/Port.php#L149

And so on.

Reply:
Looks like your suggestion is much more cleaner, I will do changes.